src/app.js

```diff
@@ -103,7 +104,8 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 document.addEventListener('copy', copySelectedToClipboard, true);
-document.addEventListener('deselectAll', function() { global.fileList.Select.deselectAll(); }, true);
+document.addEventListener('removeSelected', function() { global.fileList.Select.removeSelected(); }, true);
```

Comment:
This could also be condensed to `document.addEventListener('removeSelected', global.fileList.Select.removeSelected, true);`